### ISSUE TITLE
include ldap/ldaps support

### ISSUE THREAD

**Issue by xquery** [curl/curl-docker#53]

**Comment by xquery** [comment]
hoping to add this to next release

**Comment by missyoyo** [comment]
Hello, Seems still not see LDAP in docker,Please add it.Thank you
curl 7.87.0-DEV (x86_64-pc-linux-musl) libcurl/7.87.0-DEV OpenSSL/1.1.1s zlib/1.2.12 brotli/1.0.9 libssh2/1.10.0 nghttp2/1.47.0
Release-Date: [unreleased]
Protocols: dict file ftp ftps gopher gophers http https imap imaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp

**Comment by jamshid** [comment]
Yes please stop disabling ldap support in the images on https://hub.docker.com/r/curlimages/curl. Or maybe add an image with a "full-*" tag that contains it.

LDAP is old but still widely used and curl is a great tool for diagnosing issues with LDAP/ActiveDirectory and making queries. 

Why is everyone trying to get it out of curl? I was just complaining about RHEL/rockylinux no longer including ldap support in curl unless the package is swapped out for `curl-full`:
https://curl.se/mail/archive-2023-03/0009.html
